### PR TITLE
Update to solstice

### PR DESCRIPTION
**Pull request by 32JP**
So I finally got git working, it was mix of my internet being weird yesterday and gtigui being annoying lol.

Anyway, heres the first major update.

### CODE REVIEW

File: Solstice/bank_00.asm [hunk 1]
```diff
@@ -3339,46 +3341,49 @@ C - - - - - 0x001572 00:9562: 38        SEC
...
+- D 0 - - - 0x001585 00:9575: 0C        .byte $0C   ; X pos
+- D 0 - - - 0x001586 00:9576: 0C        .byte $0C   ; Y pos
+- D 0 - - - 0x001587 00:9577: 10        .byte $10   ; Z pos
 
+- D 0 - - - 0x001588 00:9578: EE        .byte con_room_id + $EE   ; Destination room ID
+- D 0 - - - 0x001589 00:9579: 1C        .byte $1C   ; X pos
+- D 0 - - - 0x00158A 00:957A: 1C        .byte $1C   ; Y pos
+- D 0 - - - 0x00158B 00:957B: 10        .byte $10   ; Z pos
 
-; bzk garbage? possible usage at 0x001538
```
Comment (cyneprepou4uk) [inline Solstice/bank_00.asm:3380]: restore this coment

File: Solstice/bank_00.asm [hunk 2]
```diff
@@ -4721,67 +4735,78 @@ C - - - - - 0x001D4F 00:9D3F: A9 FF     LDA #$FF
...
+;second byte is what item ID "opens" the exit, detanotars has mind of their own.
+;third byte, intial status, 00 invisible, FF visible
+;fourth byte values:
+; 01 = SW bottom-left
+; 02 = SE bottom-right
+; 04 = NE top-right
+; 08 = NW top-left
 
 tbl_9D42:
-; bzk optimize, 3rd byte is always 00
```
Comment (cyneprepou4uk) [inline Solstice/bank_00.asm:4726]: restore this comment

File: Solstice/bank_02.asm [hunk 3]
```diff
@@ -2214,7 +2233,10 @@ _off012_A6E8_02:
 
 
 
-_off012_A786_03:    ; bzk bug
```
Comment (cyneprepou4uk) [inline Solstice/bank_02.asm:2217]: restore this comment

File: Solstice/bank_02.asm [hunk 4]
```diff
@@ -2320,29 +2347,31 @@ bra_A85D_RTS:
...
-- D 1 - I - 0x01289D 04:A88D: FE        .byte $FE   ; 
-- D 1 - I - 0x01289E 04:A88E: 8D 21     .word $218D
+
+- D 1 - I - 0x01289D 04:A88D: FE        .byte $FE   ; line break
+- D 1 - I - 0x01289E 04:A88E: 8D 21     .word $218D	; position
+;10	(placeholder until its overweitten with proper value, also seems like its two digits when theres only max 8 credits?)
 ;                                              00   01   02   03   04   05   06   07   08   09   0A   0B   0C   0D   0E   0F
 - D 1 - I - 0x0128A0 04:A890: 02        .byte                                                                  $02, $01   ; 
 
-- D 1 - I - 0x0128A2 04:A892: FE        .byte $FE   ; bzk useless?
```
Comment (cyneprepou4uk) [inline Solstice/bank_02.asm:2342]: restore this comment

**Review by cyneprepou4uk — CHANGES_REQUESTED** [review]
Too many changes in bank_FF at the same time, the site doesn't load diffs (throws an error), so I can't review then. You need to make smaller updates to this file. Restore it back to original, then keep replacing a thousand lines at a time for example from your modified one.

**Comment by cyneprepou4uk** [comment]
[5eae4bd](https://github.com/cyneprepou4uk/NES-Games-Disassembly/pull/8/commits/5eae4bd6254cd8807af3a18827f32ec204600541) looks good, keep doing that for the rese of the file

**Comment by 32JP** [comment]
> [5eae4bd](https://github.com/cyneprepou4uk/NES-Games-Disassembly/pull/8/commits/5eae4bd6254cd8807af3a18827f32ec204600541) looks good, keep doing that for the rest of the file

I'll finish rest on Thursday at earliest, or by weekend.

**Comment by 32JP** [comment]
Okay this should be fine now, Im done for now unless something comes up

File: Solstice/bank_FF.asm [hunk 5]
```diff
@@ -9819,91 +10024,117 @@ _off000_A2A2_44:
...
+- D 1 - I - 0x01A31E 06:A30E: 00        .byte $00   ; Z position
+- D 1 - I - 0x01A31F 06:A30F: 80        .byte $80   ; tile setting
+;special blocks
+- D 1 - I - 0x01A320 06:A310: 00        .byte $00   ; how many
+
+
+;garbage, 222 bytes of free space!
+;kinda looks like block data, but it makes zero sense.
 
-; bzk garbage, unused room?
```
Comment (cyneprepou4uk) [inline Solstice/bank_FF.asm:9901]: restore this comment

**Comment by cyneprepou4uk** [comment]
Great! Thank you for your contribution, and your patience :)

**Pull request merged**